src/components/audio-breakdown-drawer/AudioBreakdownDrawer.tsx

```diff
@@ -316,7 +317,7 @@ const Wallet = ({ chain, address, balance }: WalletProps) => {
   const solWalletAudioEnabled = false
   const styles = useThemedStyles(createStyles)
 
-  const scale = useRef(new Animated.Value(1)).current
+  const { scale, startPress, stopPress } = usePressScaleAnimation(0.98)
```

Comment:
might be nice to have these `{ scale, onPressIn, onPressOut }` so it's super obvious where they should go